.gitignore

```diff
@@ -1,8 +1,7 @@
 /.vagrant/
```

Comment:
All traces of Vagrant can be removed in my opinion. But feel free to do this in a new PR!

Reply:
@quartje confirmed that ansible / vagrant can be removed from the premises.